<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetCourbes</class>
 <widget class="QWidget" name="WidgetCourbes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Courbes" name="widgetX" native="true">
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>405</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(220,220,220), stop: 1 rgb(180,180,180));

border-color: rgb(240,240,240);
border-radius: 6px;</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(220,220,220), stop: 1 rgb(213,213,213));</string>
       </property>
       <property name="text">
        <string>Axe des X</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Courbes" name="widgetY" native="true">
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>405</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(220,220,220), stop: 1 rgb(180,180,180));

border-color: rgb(240,240,240);
border-radius: 6px;</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(220,220,220), stop: 1 rgb(213,213,213));</string>
       </property>
       <property name="text">
        <string>Axe des Y</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Courbes" name="widgetZ" native="true">
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>405</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(220,220,220), stop: 1 rgb(180,180,180));

border-color: rgb(240,240,240);
border-radius: 6px;</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(220,220,220), stop: 1 rgb(213,213,213));</string>
       </property>
       <property name="text">
        <string>Axe des Z</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Courbes</class>
   <extends>QWidget</extends>
   <header>view/courbes.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
